XCUITests/FxScreenGraph.swift

```diff
@@ -990,10 +991,14 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         screenState.tap(app.buttons["HomePanels.Downloads"], to: HomePanel_Downloads)
     }
 
+    map.addScreenState(LoginsBrowserTab) { screenState in
```

Comment:
I think there is no need to create a new screenState...we already have this one to go to Logins...
-When there is[ no password set](https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/FxScreenGraph.swift#L771)
-When there is [password set](https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/FxScreenGraph.swift#L775)